server/base/src/main/java/org/apache/accumulo/server/fs/VolumeChooserEnvironment.java

```diff
@@ -65,7 +65,7 @@ public Text getEndRow() {
     return endRow;
   }
 
-  public Table.ID getTableId() {
+  public TableId getTableId() {
```

Comment:
In 1.9 this returned a string.

Reply:
This PR didn't make that change. It just made the change from an internal type to a public API type.

Reply:
yeah I know it didn't make the change... its just something I noticed.  The change was made in the switch to Table IDs